project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.6
+sbt.version=0.13.16
```

Comment:
Can this stay at 0.13.6?  The goal is to keep track of the earliest working version so we can [document it](https://github.com/earldouglas/xsbt-web-plugin/blob/9998757a23c1b5f3b090be56f8c4c1d5d953011d/README.md#requirements).